notes/ltc-transfer-penalty.vbs

```diff
@@ -241,4 +256,51 @@ Call write_variable_in_case_note (row_9)
 Call write_variable_in_case_note ("---")
 call write_variable_in_case_note (worker_signature)
 
-script_end_procedure ("Success, your penalty case note has been created.  Please ensure that your notices also reflect the transfer information.")							'closing script and writing stats
\ No newline at end of file
+script_end_procedure ("Success, your penalty case note has been created.  Please ensure that your notices also reflect the transfer information.")							'closing script and writing stats
+
+
+
+'----------------------------------------------------------------------------------------------------Closing Project Documentation - Version date 01/12/2023
```

Comment:
There's a more recent Closing Issue Documentation version at: MAXIS-scripts/Closing Issue Documentation.vbs. Please update the extra elements and resubmit.

Reply:
You were right, that was my bad. Resolved!